### ISSUE TITLE
Finder integration breaks on macOS 12.0.1

### ISSUE THREAD

**Issue by dnmvisser** [keybase/client#24668]
Upgraded from macOS 11.16 to 12.0.1 last night. It is 'normal' for OS upgrades to cause the Keybase finder integration to break, which then always requires manual fixing. But this time around that is not possible anymore. As it stands now, finder integration is broken entirely (for me, at least).
This is `Version 5.8.0-20210920184218+ac2177aec7 (5.8.0-20210920184218+ac2177aec7)` on a MacBook Pro (13-inch, 2020, Four Thunderbolt 3 ports).

Observations:

1. In the main app => Settings => Finder integration, clicking the `Enable Finder integration` will bring up the `Enable Keybase in Finder?` dialog, and I tick the

   > I understand that a closed-source kernel extension (FUSE for macOS) will be installed.

   box and click `Yes, enable`. Green button dims and spinner starts, 10 seconds later green button is back again. This can be repeated indefinitely. Rebooting and approving kexts does not result in working Finder integration.

1. There is a list of different kernel extensions, with what seems to be symlinks for different OS releases. This lack a `12`-ish version currently:
    ```
    dnmvisser@GA0267 ~$ ls -la /Library/Filesystems/kbfuse.fs/Contents/Extensions/
    total 0
    drwxr-xr-x@ 6 root  wheel  192 20 Sep 20:54 .
    drwxr-xr-x@ 7 root  wheel  224 20 Sep 20:54 ..
    drwxr-xr-x@ 3 root  wheel   96 20 Sep 20:54 10.14
    lrwxr-xr-x  1 root  wheel    5 20 Sep 20:54 10.15 -> 10.14
    lrwxr-xr-x  1 root  wheel    2 20 Sep 20:54 10.16 -> 11
    drwxr-xr-x@ 3 root  wheel   96 20 Sep 20:54 11
    ```
    Manually adding a symlink from `12` to `11` doesn't work because as soon as the Keybase app starts, the directory is wiped and repopulated.

1. Manually loading the kernel extension requires approval (which is expected):

    ```
    dnmvisser@GA0267 ~$ sudo kextutil -l /Library/Filesystems/kbfuse.fs/Contents/Extensions/11/kbfuse.kext
    Password:
    Executing: /usr/bin/kmutil load --bundle-path /Library/Filesystems/kbfuse.fs/Contents/Extensions/11/kbfuse.kext --load-style start-only
    Error Domain=KMErrorDomain Code=27 "Extension with identifiers com.github.kbfuse.filesystems.kbfuse,org.virtualbox.kext.VBoxDrv,org.virtualbox.kext.VBoxNetAdp,org.virtualbox.kext.VBoxUSB,org.virtualbox.kext.VBoxNetFlt not approved to load. Please approve using System Preferences." UserInfo={NSLocalizedDescription=Extension with identifiers com.github.kbfuse.filesystems.kbfuse,org.virtualbox.kext.VBoxDrv,org.virtualbox.kext.VBoxNetAdp,org.virtualbox.kext.VBoxUSB,org.virtualbox.kext.VBoxNetFlt not approved to load. Please approve using System Preferences.}
    ```

    But then after a reboot, loading it fails:

    ```
    dnmvisser@GA0267 ~$ sudo kextutil -l /Library/Filesystems/kbfuse.fs/Contents/Extensions/11/kbfuse.kext
    Password:
    Executing: /usr/bin/kmutil load --bundle-path /Library/Filesystems/kbfuse.fs/Contents/Extensions/11/kbfuse.kext --load-style start-only
    Error Domain=KMErrorDomain Code=71 "Kernel request failed: (libkern/kext) kext (kmod) start/stop routine failed (-603946985)" UserInfo={NSLocalizedDescription=Kernel request failed: (libkern/kext) kext (kmod) start/stop routine failed (-603946985)}
    ```

**Comment by jeromelaurens** [comment]
same here

**Comment by RayOei** [comment]
Please 'vote' on main issue instead of adding "me too's"!

**Comment by JPry** [comment]
@RayOei Which is the main issue? #24578?

**Comment by RayOei** [comment]
@JPry The main post or indeed the first report of the same issue. Adding "me too's" is not helpful for the maintainers, generates a lot of noise with the email notifications and doesn't add new information to an issue. That is where the voting (emojis) is for.

**Comment by maxtaco** [comment]
Closed as duplicative of #24636

**Comment by Groggy66** [comment]
I'm sorry for any mistakes 👋
